WeScan/Review/ReviewViewController.swift

```diff
@@ -126,15 +126,14 @@ final class ReviewViewController: UIViewController {
     @objc private func toggleEnhancedImage() {
         guard enhancedImageIsAvailable else { return }
         
+        isCurrentlyDisplayingEnhancedImage.toggle()
         reloadImage()
         
         if isCurrentlyDisplayingEnhancedImage {
-            enhanceButton.tintColor = .white
-        } else {
             enhanceButton.tintColor = UIColor(red: 64 / 255, green: 159 / 255, blue: 255 / 255, alpha: 1.0)
+        } else {
```

Comment:
Can we maybe write a test to reproduce this issue how it was, to make sure we don't introduce it again?

Reply:
This is a good idea.